modules/ocf_tv/files/i3/config

```diff
@@ -148,6 +153,32 @@ mode "resize" {
 
 bindsym $mod+r mode "resize"
 
+# Volume Control
```

Comment:
it may be better to do this the same way `ocf-tv` does it? https://github.com/ocf/utils/blob/87cbb68404c72bd21ce0b908e4b71303f1a34909/staff/lab/ocf-tv#L117